dts/bindings/auxdisplay/sparkfun,serlcd.yaml

```diff
@@ -32,3 +34,13 @@ properties:
     enum:
       - 2
       - 4
+
+  command-delay:
+    type: int
+    default: 10
+    description: "Delay in milliseconds after a normal command was sent"
+
+  special-command-delay:
+    type: int
+    default: 50
+    description: "Delay in milliseconds after a special command was sent"
```

Comment:
The quote's are not needed, and defaults need to be justified

Reply:
removed the quote signs.
but regarding the default values, I kept the exact same indentation as the original code, can you please explain what kind of justification you mean?

![image](https://github.com/zephyrproject-rtos/zephyr/assets/700985/60dbf0a1-b1ac-458a-ae8f-2a5e5ff7f3aa)

Reply:
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values

Reply:
@thedjnK got it - thanks for the example.

It is better now?
To include the actual link to the SerLCD Sparkfun library?

Reply:
Great, thanks